projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -31,26 +30,22 @@ export class PageTimeRangePreferenceService {
     this.pageTimeRangeStringDictionary$ = this.buildPageTimeRangeObservable();
   }
 
-  public getTimeRangePreferenceForPage(rootLevelPath: string): Observable<TimeRange> {
+  public getTimeRangePreferenceForPage(rootLevelPath: string): Observable<TimeRangeCallback> {
     return combineLatest([
       this.pageTimeRangeStringDictionary$,
-      this.featureStateResolver.getCombinedFeatureState([
-        ApplicationFeature.PageTimeRange,
-        ApplicationFeature.NavigationRedesign
-      ])
+      this.featureStateResolver.getFeatureState(ApplicationFeature.PageTimeRange)
     ]).pipe(
       map(([pageTimeRangeStringDictionary, featureState]) => {
         if (featureState === FeatureState.Enabled) {
           if (isNil(pageTimeRangeStringDictionary[rootLevelPath])) {
-            // Right side for when FFs are enabled but page path doesn't have 'defaultTimeRange' set on AR data
-            return this.getDefaultPageTimeRange() ?? this.globalDefaultTimeRange;
+            return this.getDefaultTimeRangeForCurrentRoute;
           }
 
-          return this.timeRangeService.timeRangeFromUrlString(pageTimeRangeStringDictionary[rootLevelPath]);
+          return () => this.timeRangeService.timeRangeFromUrlString(pageTimeRangeStringDictionary[rootLevelPath]);
         }
 
-        // When FFs are disabled
-        return this.globalDefaultTimeRange;
+        // When FF is disabled
+        return this.getGlobalDefaultTimeRange;
```

Comment:
nit: use fat arrow here - assigning a function an instance member is more confusing for a reader, because when looking at this line, the reader has to go verify how `this.getGlobalDefaultTimeRange` was written; when looking at the defining line, the question is raised - why is this not written as a normal prototype function? That is, we're coupling two lines of code unncessarily.

Reply:
fixed